ts/Dashboards/Layout/Cell.ts

```diff
@@ -289,7 +289,7 @@ class Cell extends GUIElement {
             json.options.parentElement = cell.id;
         }
 
-        const component = componentFromJSON(json, cellContainer);
+        const component = componentFromJSON(json);
```

Comment:
GitHub Actions report an unused variable on line 284. I think it is bad practice leaving unimplemented code with a `// @todo` comment. Mostly `@todo`s like this get forgotten about anyway. So I think we should remove the business code, and either keep a note somewhere else, or leave it commented out.